storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -134,7 +134,7 @@ bool Redo_Log_Reader::is_error() const { return (m_error); }
 @param[in,out]  desired_lsn             LSN that triggered file reopening.
 @return true if re-open was sucessful */
 
-static bool reopen_log_files(lsn_t desired_lsn) {
+static bool reopen_log_files(lsn_t desired_lsn = 0) {
```

Comment:
please remove the default parameter. Better to pass explicitly.

Reply:
fixed.